src/builtins/math.rs

```diff
@@ -1,17 +1,29 @@
+use widestring::utf32str;
+
 use super::prelude::*;
 use crate::tinyexpr::te_interp;
 
 /// The maximum number of points after the decimal that we'll print.
 const DEFAULT_SCALE: usize = 6;
 
+const DEFAULT_ZERO_SCALE_MODE: ZeroScaleMode = ZeroScaleMode::TRUNC;
+
 /// The end of the range such that every integer is representable as a double.
 /// i.e. this is the first value such that x + 1 == x (or == x + 2, depending on rounding mode).
 const MAX_CONTIGUOUS_INTEGER: f64 = (1_u64 << f64::MANTISSA_DIGITS) as f64;
 
+enum ZeroScaleMode {
+    TRUNC,
+    ROUND,
+    FLOOR,
+    CEIL,
+}
+
```

Comment:
I know we don't (yet) have a consistent naming policy across the crate, but we are trying to adopt rust conventions especially for new code. Could you please change these to be idiomatic (e.g. `ZeroScaleMode::Trunc`), and the same for any other all-caps non-`const` values/variables?

Reply:
Got it